docs/user-guide/concepts/model-providers/amazon-bedrock.md

```diff
@@ -16,119 +16,30 @@ The [`BedrockModel`](../../../api-reference/models.md#strands.models.bedrock) cl
 
 1. **AWS Account**: You need an AWS account with access to Amazon Bedrock
 2. **Model Access**: Request access to your desired models in the Amazon Bedrock console
-3. **AWS Credentials**: Configure AWS credentials with appropriate permissions, including `bedrock:InvokeModelWithResponseStream`
+3. **AWS Credentials**: Configure AWS credentials with appropriate permissions
 
 #### Required IAM Permissions
 
 To use Amazon Bedrock with Strands, your IAM user or role needs the following permissions:
 
-- `bedrock:InvokeModelWithResponseStream`
+- `bedrock-runtime:InvokeModelWithResponseStream` (for streaming mode)
+- `bedrock-runtime:InvokeModel` (for non-streaming mode)
 
 Here's a sample IAM policy that grants the necessary permissions:
 
 ```json
 {
-  "Version": "2012-10-17",
-  "Statement": [
-    {
-      "Effect": "Allow",
-      "Action": ["bedrock:InvokeModelWithResponseStream"],
-      "Resource": "*"
-    }
-  ]
-}
-```
-
-For production environments, it's recommended to scope down the `Resource` to specific model ARNs.
-
-#### Requesting Access to Bedrock Models
```

Comment:
Let's keep this section - we've had a lot of questions about this when getting started with bedrock models

Reply:
Or was this section duplicated?